_glaredb/sql-commands/create-credentials.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: CREATE CREDENTIALS
+parent: SQL commands
+---
+
+# CREATE CREDENTIALS
+
+Create credentials for accessing AWS and Google Cloud resources.
+
+## Syntax
+
+```sql
+CREATE CREDENTIALS <credential-name>
+    PROVIDER <provider>
+    OPTIONS (<provider-options>);
+```
+
+| Field              | Description                                                       |
+| ------------------ | ----------------------------------------------------------------- |
+| `credential-name`  | Name for the credentials.                                         |
+| `provider`         | The cloud provider for these credentials are for. `aws` or `gcp`. |
+| `provider-options` | Provider specific options. See below sections.                    |
+
+## AWS credentials
+
+AWS credentials allows GlareDB to read and write objects in S3.
+
+Creating AWS credentials requires `access_key_id` and `secret_access_key`. These
+correspond to an IAM user with permissions for accessing objects in S3.
+
+```sql
+CREATE CREDENTIALS my_aws_creds
+    PROVIDER aws
+    OPTIONS (
+        access_key_id = 'my_access_key_id',
+        secret_access_key = 'my_secret_access_key',
+    );
+```
+
+We can use the above credentials to access objects in S3:
```

Comment:
I find `above` to be sort of the documentation equivalent of a raw pointer: easy to break, hard to detect except at runtime, sometimes. 

Better I think to describe the state: ("After configuring the credentials"